pico/api/src/main/java/module-info.java

```diff
@@ -17,18 +17,17 @@
 /**
  * Pico API module.
  */
-module io.helidon.pico {
+module io.helidon.pico.api {
+    requires jakarta.inject;
+    requires io.helidon.common.types;
     requires io.helidon.common;
     requires io.helidon.common.config;
-    requires io.helidon.pico.types;
     requires static io.helidon.builder;
     requires static io.helidon.config.metadata;
-
     requires static jakarta.annotation;
-    requires jakarta.inject;
+
+    uses io.helidon.pico.spi.PicoServicesProvider;
```

Comment:
Why did you change the order? We usually have `exports` before `uses`

Reply:
pushed